docs/get-started.md

```diff
@@ -0,0 +1,114 @@
+## Overview
+
+This document describes configuration of the forked Helm Broker bundles repository.
+
+## Prerequisites
+
+Before you start install:
+
+* [Kyma](https://kyma-project.io/docs/latest/root/kyma#getting-started-getting-started) in version v0.4.1 or later
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in version v1.10.0 or later
+* [svcat CLI](https://svc-cat.io/docs/cli/) in version v1.10.0 or later
+
+## Steps
+
+### Create fork
+
+1. Create fork from the [bundle repository](https://github.com/kyma-project/bundles)
+
+2. On your forked repository create `gh-pages` branch if not exist
+
+3. Go to **Settings** tab and in section **GitHub Pages** setup **Source** to `gh-pages branch`
+
+### Integrate Travis-CI with Github repository
+
+1. Go to [Github](https://github.com) page.
+
+2. Choose **Settings** tab on your [Github profile account](https://github.com/settings/profile), in the left sidebar, click **Developer settings**.
+
+3. In the left sidebar, click **Personal access tokens**.
+
+4. Click **Generate new token**.
+
+5. Give your token a descriptive name.
+
+6. Select scope `repo -> public_repo` (Access public repository)
+
+7. Click **Generate token** and copy your personal access token.
+
+8. Go to [travis-ci](https://travis-ci.com/) and login using your Github account
+
+9. Add access to your forked `bundle` repository.
+
+10. In the left sidebar, click **My Repositories** tab and choose your Github repository
+
+11. In the right tab click **More options** then **Settings**
+
+12. Go to **Environment Variables** section and add key-value pair where key is `GITHUB_TOKEN` and value is equal to your Github personal access token.
+
+13. In the right tab click **More options** then **Trigger build** and wait for Travis to finish his work
+
+### Create bundle
+
+1. Create your bundle according to [instruction](https://kyma-project.io/docs/latest/components/service-brokers#configuration-how-to-create-a-ybundle)
+
+2. Place your bundle into the `bundle/{repository}` directory.
+
+3. Along your bundle directory create `index.yaml` which defines available bundles. 
+Instruction how to create a valid file can be found [here](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/011-configuration-helm-broker.md#configuring-the-helm-broker-externally)
+
+4. Commit changes.
```

Comment:
"Commit changes" + " and push to the master branch" 

because of the configuration in .travis.yaml file - Travis is deploying bundles on gh-pages only when the new commit is pushed to master. Of course, this behaviour it can be changed int **.travis.yaml** file, so we can add such _note_